gnd/src/main/java/com/google/android/gnd/util/GeoJsonParser.java

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
A few suggestions on code structure:
- [ ] This deals with deserializing JSON files, so it probably in the persistence package. (e.g. `...persistence.geojson?`)
- [ ] Rather than pass JSONObject around to static methods, can we wrap the different instances in typed accessor objects? (e.g., JsonGeometry, JsonTileExtent, etc)? (https://testing.googleblog.com/2017/11/obsessed-with-primitives.html)
- [ ] If this file is  moved to its own package, the inner classes and be promoted to outer classes in their own file to help readability.
- [ ] GeoJsonParser can then provide a custom GeoJson object with the data we're interested in.

Wdyt?

Reply:
SG! After the initial impl, this component seems complex enough to me to up-level it to its own package, which will make it easier to define accessor objects like you suggest--plus it'll make it easier to extend in the future.

Reply:
Thanks for taking a look!